src/plugins/intel_cpu/src/nodes/scatter_update.cpp

```diff
@@ -198,34 +217,15 @@ void ScatterUpdate::initSupportedPrimitiveDescriptors() {
     dataPrec = getOriginalInputPrecisionAtPort(DATA_ID);
     dataSize = dataPrec.size();
 
-    bool canBeInplace = !isDynamicNode() && getParentEdgeAt(DATA_ID)->getParent()->getChildEdges().size() == 1 &&
-                        !getParentEdgeAt(DATA_ID)->getParent()->isConstant();
+    bool canBeInplace = !getParentEdgeAt(DATA_ID)->getParent()->isConstant();
 
-    NodeConfig config;
-    if (axisRelaxed) {
-        config.inConfs.resize(4);
-    } else {
-        config.inConfs.resize(3);
-    }
-    config.outConfs.resize(1);
-    config.inConfs[DATA_ID].constant(false);
-    config.inConfs[INDICES_ID].constant(false);
-    config.inConfs[UPDATE_ID].constant(false);
-    config.outConfs[0].constant(false);
-    config.inConfs[DATA_ID].inPlace(canBeInplace ? 0 : -1);
-    config.inConfs[INDICES_ID].inPlace(-1);
-    config.inConfs[UPDATE_ID].inPlace(-1);
-    config.outConfs[0].inPlace(canBeInplace ? 0 : -1);
-    if (axisRelaxed) {
-        config.inConfs[AXIS_ID].constant(false);
-        config.inConfs[AXIS_ID].inPlace(-1);
-    }
-
-    std::vector<PortConfigurator> inPortConfig{{LayoutType::ncsp, dataPrec}, {LayoutType::ncsp, indicesPrec}, {LayoutType::ncsp, dataPrec}};
+    std::vector<PortConfigurator> inPortConfig{{LayoutType::ncsp, dataPrec, false, canBeInplace ? 0 : -1},
+                                                {LayoutType::ncsp, indicesPrec},
+                                                {LayoutType::ncsp, dataPrec}};
```

Comment:
@maxnick Do you see any problem here to make SEU node inplaced?

Reply:
I don't expect any problems here.